ivy/functional/frontends/jax/numpy/manipulations.py

```diff
@@ -133,6 +133,13 @@ def atleast_2d(*arys):
     return ivy.atleast_2d(*arys)
 
 
+@to_ivy_arrays_and_back
+def block(arr, block_size):
+    if isinstance(arr, ivy.Array):
+        arr_blocks = ivy.reshape(arr, ivy.concat([ivy.shape(arr)[:-1], [-1, block_size]], 0))
+        return arr_blocks
```

Comment:
@Daniel4078 sir is this one fine?

Reply:
Looks fine to me. Let's wait and see the result of the automatic checks.